### ISSUE TITLE
Mask loss with soft labels

### ISSUE THREAD

**Issue by antopost** [facebookresearch/Mask2Former#236]
Hey @bowenc0221 
I was wondering if in theory it would be possible to train a model that uses mask loss with soft label ground truth.
If so, would any alterations have to be done to the code or could I pass the prediction and soft labels to the loss function as is?
